### PR TITLE
fixed gvcf2hvcf documentation

### PR DESCRIPTION
**Pull request by lynnjo**
## Description

_Provide a summary of your changes including motivation and context.
If these changes fix a bug or resolves a feature request, be sure to link to that issue._

## Type of change

_What type of changes does your code introduce? Put an `x` in boxes that apply._

- [ ] `CHANGE` (fix or feature that would cause existing functionality to not work as expected)
- [ ] `FEATURE` (non-breaking change which adds functionality)
- [ ] `BUGFIX` (non-breaking change which fixes an issue)
- [ ] `ENHANCEMENT` (non-breaking change which improves existing functionality)
- [x] `NONE` (if none of the other choices apply. Example, tooling, build system, CI, docs, etc.)

## Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have updated relevant documentation

## Changelog entry

_Please add a one-line changelog entry below. This will be copied to the changelog file during the release process._

```release-note
Fixed documentation for gvcf2hvcf - corrected the parameter definitions.
```

### CODE REVIEW

File: docs/build_and_load.md [hunk 1]
```diff
@@ -823,14 +822,22 @@ phg create-maf-vcf \
     --maf-dir output/alignment_files \
     -o output/vcf_files
 ```
-
+3, Create hVCF from existing PHG created gVCF files
+```shell
+phg gvcf2hvcf \
```
Comment (tcasstevens) [inline docs/build_and_load.md:827]: Should this have the same parameter values as above for consistency?

**Review by tcasstevens — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -63,7 +63,7 @@ _Long-form documentation for this section can be found [here](docs/build_and_loa
 ./phg create-maf-vcf --db-path /path/to/dbs --bed /my/bed/file.bed --reference-file /my/ref.fasta --maf-dir /my/maf/files -o /path/to/vcfs
 
 ## Convert GVCF to HVCF: use this if you have GVCF files created by PHG, but do not have MAF or h.vcf files
```
Comment (zrm22) [inline README.md:65]: This header should say Optional or instead of using create-maf-vcf.

**Review by zrm22 — APPROVED** [review]

File: docs/build_and_load.md (hunk 1 above)
Comment (lynnjo) [reply docs/build_and_load.md:827]: @tcasstevens - which above parameters?  Do you mean the general example on line 73, or the more detailed example for create-maf-vcf directly above?  I basically followed what was shown for create-maf-vcf above, but the gvcf folder is different as it is on the user had previously.

I'm happy to change this however you want, I'm just not clear on what you are recommending.

File: README.md (hunk 2 above)
Comment (lynnjo) [reply README.md:65]: changed.

File: docs/build_and_load.md [hunk 3]
```diff
@@ -823,14 +822,22 @@ phg create-maf-vcf \
     --maf-dir output/alignment_files \
     -o output/vcf_files
 ```
-
+3, Create hVCF from existing PHG created gVCF files
```
[the remark this answers is not in the document]
Comment (lynnjo) [reply docs/build_and_load.md:825]: Updated.

**Pull request merged**